keps/sig-storage/20190408-volume-scheduling-limits.md

```diff
@@ -248,13 +249,13 @@ In-tree volume plugins are being migrated to CSI in a [separate KEP](https://git
 
 * CSI migration feature is `off` for a volume plugin and CSI driver is not installed on a node:
   * In-tree volume plugin is used for in-tree PVs.
-  * Kubelet gets volume limits from the in-tree volume plugin and creates `CSINode` during node registration. Scheduler and kubelet still needs to translate in-tree plugin name to CSI driver name to get the right `name` for `CSINode.spec.drivers[xyz].allocatable`.
+  * Kubelet gets volume limits from the `Node` object and existing predicate do their work.
 
 * CSI migration feature is `off` for a volume plugin and CSI driver (for the same storage backend) is installed on a node:
   * In-tree volume plugin is used for in-tree PVs.
   * CSI driver is used for CSI PVs.
-  * Kubelet gets volume limits from the in-tree volume plugin and creates `CSINode` during node registration.
-  * When CSI driver is registered, kubelet gets its volume limit through CSI and updates `CSINode`, potentially overwriting in-tree plugin limit.
+  * Kubelet gets volume limits from the in-tree volume plugin and updates `Node` object with limit. For in-tree volumes existing predicate to their work.
+  * When CSI driver is registered, kubelet gets its volume limit through CSI and updates `CSINode`. At this point existing predicate for in-tree plugin becomes NOOP and both in-tree PVs and CSI PVs are counted by `MaxCSIVolumeCountPred` predicate.
```

Comment:
This means that even if migration is off, we're still doing the translation in case the CSI equivalent is installed.

Reply:
Yes that is true.  Is there a downside I am missing?

Reply:
No, I think it would be good to call it out though because it's not totally obvious from reading it

Reply:
fixed